pytradfri/const.py

```diff
@@ -56,8 +56,8 @@
 ATTR_LIGHT_COLOR_HEX = "5706"  # string representing a value in hex
 ATTR_LIGHT_COLOR_X = "5709"
 ATTR_LIGHT_COLOR_Y = "5710"
-ATTR_LIGHT_COLOR_HUE = "5707"
-ATTR_LIGHT_COLOR_SATURATION = "5708"
+ATTR_LIGHT_COLOR_HUE = "5708"
```

Comment:
Are you certain about this? Looking at the constants in the Android app, it should be the other way around: https://hastebin.com/zofoqucibe.php

Reply:
You are right, I've changed a bit too much here. The other places I flipped hue with sat are correct though.